Tweets.py

```diff
@@ -285,33 +255,35 @@ def on_data(self, data):
                         break
                     else:
                         return True
-                # TODO: http://vk.cc/5raROT
-                geo_tweet = '\n\t{ "type": "Geotweet", \n\t\t"geometry":\n\t\t\t'
+
+                geo_tweet = collections.OrderedDict({})
+                geo_tweet['type'] = 'Geotweet'
+
                 if not jdata['geo']:  # Place
                     # Change Polygon to Point
                     coordinates = jdata['place']['bounding_box']['coordinates'][0]
                     lon = (coordinates[0][0] + coordinates[2][0]) / 2
                     lat = (coordinates[0][1] + coordinates[2][1]) / 2
-                    geo_tweet += '{"type": "Point", "coordinates": [' + str(lon) + ', ' + str(lat) + ']}'
+                    geo_tweet['geometry'] = {'type': 'Point', 'coordinates': [lon, lat]}
                 else:  # Geo
-                    geo_tweet += str(jdata['geo']).replace("u'", '"').replace("'", '"')
+                    geo_tweet['geometry'] = jdata['geo']
 
                 # Add metadata which user choose
-                geo_tweet += ',\n\t\t"properties": \n\t\t\t'
-                if output_metadata == 'Minimum (created_at, text, coordinates)':
-                    min_data = '"created_at":"' + str(jdata['created_at']) + '",' + \
-                                '"text":"' + str(jdata['text'].encode('utf-8'))\
-                                .replace('"', "'").replace('\\', '\\\\') + '"'  # crutch-escape characters
-                    geo_tweet += '{' + min_data + '}\n'
-                elif output_metadata == 'All (need more RAM / big files)':
-                    geo_tweet += data
-                elif output_metadata == 'None - Only coordinates':
-                    geo_tweet += '{}'
-                geo_tweet += '\t}'
+                if output_metadata == 'min':
+                    min_data = collections.OrderedDict({})
+                    min_data['created_at'] = str(jdata['created_at'])
+                    min_data['text'] = jdata['text']
+                    geo_tweet['properties'] = min_data
+                elif output_metadata == 'all':
+                    geo_tweet['properties'] = jdata
+
+                elif output_metadata == 'none':
+                    geo_tweet['properties'] = {}
 
                 if output_GeoJSON:
                     with open(file_name + '.geojson', 'a') as file:
```

Comment:
Never mask `file`. It's a built-in:
```
In [5]: file?
Docstring:
file(name[, mode[, buffering]]) -> file object

Open a file.  The mode can be 'r', 'w' or 'a' for reading (default),
writing or appending.  The file will be created if it doesn't exist
when opened for writing or appending; it will be truncated when
opened for writing.  Add a 'b' to the mode for binary files.
Add a '+' to the mode to allow simultaneous reading and writing.
If the buffering argument is given, 0 means unbuffered, 1 means line
buffered, and larger numbers specify the buffer size.  The preferred way
to open a file is with the builtin open() function.
Add a 'U' to mode to open the file for input with universal newline
support.  Any line ending in the input file will be seen as a '\n'
in Python.  Also, a file so opened gains the attribute 'newlines';
the value for this attribute is one of None (no newline read yet),
'\r', '\n', '\r\n' or a tuple containing all the newline types seen.

'U' cannot be combined with 'w' or '+' mode.
Type:      type

In [6]: open?
Docstring:
open(name[, mode[, buffering]]) -> file object

Open a file using the file() type, returns a file object.  This is the
preferred way to open a file.  See file.__doc__ for further information.
Type:      builtin_function_or_method
```

Use `f` or `file_` instead